core/arch/arm/plat-imx/system_config.mk

```diff
@@ -0,0 +1,19 @@
+# this script defines platform specific settings for TEE armv7 build
+ifeq ($(PLATFORM_FLAVOR),mx6ulevk)
+
+# Static configuration of DDR reserved to TEE/TZ
+# Provide here only start address and size. TEEcore will adapt.
+# Allow these settings to be overridden.
+CFG_DDR_TEETZ_RESERVED_START ?= 0x9c100000
```

Comment:
This file wouldn't be needed if you provided the information in `platform_config.h` instead. Some platform still uses  a file like this, but that's mostly for legacy reasons.